src/coreclr/src/jit/emit.cpp

```diff
@@ -65,6 +65,21 @@ UNATIVE_OFFSET emitLocation::GetFuncletPrologOffset(emitter* emit) const
     return emit->emitCurIGsize;
 }
 
+//------------------------------------------------------------------------
+// IsPreviousIns: Returns true if the emitter is on the next instruction
+//  of the same group than this emitLocation.
+//
+// Arguments:
+//  emit - an emitter* instance
+//
+bool emitLocation::IsPreviousIns(const emitter* emit) const
+{
+    assert(Valid());
+    bool sameGroup  = ig == emit->emitCurIG;
+    bool sameInsNum = emitGetInsNumFromCodePos(codePos) == emitGetInsNumFromCodePos(emit->emitCurOffset()) - 1;
```

Comment:
I would rename this as `isPreviousInsNum` as it's not actually the same.
And I would rename `sameGroup` as `isSameGroup` (`sameGroup` could mean the actual group number).